romana/cmd/helpers.go

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2016 Pani Networks
+// All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations
+// under the License.
+
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
```

Comment:
Another one of those fancy names! :-)

So, what is "cobra" doing? I have no idea. What's a "cobra.Command"?

Should I really have to google for every imported, minor but grandiosely named utility library someone is using when reading code? I think a small comment would be helpful. Maybe just a few words behind the import line (or before each import line), giving a quick heads up on what an imported package is about.